src/views/puzzles/PuzzleView/PuzzleView.vue

```diff
@@ -97,22 +92,22 @@
       TagsPanel,
       SidebarPanel,
       Preloader,
-      Comments
+      Comments,
     },
   })
   export default class PuzzleView extends Mixins(PageDataMixin(fetchPageData)) {
     private puzzleRoute: string = PUZZLE_ROUTE_PREFIX;
 
     get puzzle() {
-      return get(this.pageData, 'puzzle');
+      return this.pageData?.puzzle;
     }
 
     get imageURL() {
-      return Utils.getPuzzleMiddleThumbnail(get(this.pageData, 'nid', ''));
+      return Utils.getPuzzleMiddleThumbnail(this.pageData?.nid as string);
     }
 
     get avatar() {
-      return Utils.getAvatar(this.puzzle.userpicture);
+      return Utils.getAvatar(this.puzzle?.userpicture as string);
```

Comment:
I am pretty sure this can return null (or undefined, idk); if so change to `as string?`

Reply:
changed